extensions/amp-access/amp-access.md

```diff
@@ -25,7 +30,7 @@ limitations under the License.
   </tr>
   <tr>
     <td class="col-fourty"><strong>Availability</strong></td>
-    <td>Stable</td>
+    <td>In development</td>
```

Comment:
@bpaduch Yes. I think we should just remove experimental notes. The spec version should be 0.1.